soh/soh/OTRGlobals.cpp

```diff
@@ -1718,6 +1733,19 @@ Color_RGB8 GetColorForControllerLED() {
                     break;
             }
         }
+        if (gPlayState && (source == LED_SOURCE_NAVI)) {
+            Actor* arrowPointedActor = gPlayState->actorCtx.targetCtx.arrowPointedActor;
+            if (arrowPointedActor) {
+                uint8_t category = gPlayState->actorCtx.targetCtx.arrowPointedActor->category;
+                color.r = sNaviColorList[category].inner.r;
+                color.g = sNaviColorList[category].inner.g;
+                color.b = sNaviColorList[category].inner.b;
+            } else {
+                color.r = sNaviColorList[2].inner.r;
+                color.g = sNaviColorList[2].inner.g;
+                color.b = sNaviColorList[2].inner.b;
+            }
```

Comment:
Or possibly make custom cosmetic it's own option like with the tunic options

Reply:
Thank you for your suggestion! I was indeed planning on adding that but you helped a ton in pointing me in the right direction.

I wonder though, if it could be done even tidier if instead of a switch statement, it used an array of CVar names.

Reply:
Yeah I think there may be a way but it would always end up being much less readable than it is now